frontend/public/components/cluster-service-broker.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-undef */
+
+import * as React from 'react';
+import { ColHead, DetailsPage, List, ListHeader, ListPage, ResourceRow } from './factory';
+import { Cog, SectionHeading, detailsPage, navFactory, ResourceLink, ResourceCog, ResourceSummary, StatusWithIcon, Timestamp } from './utils';
+// eslint-disable-next-line no-unused-vars
+import { K8sResourceKind, K8sResourceKindReference } from '../module/k8s';
+import { Conditions } from './conditions';
+import { ClusterServiceClassPage } from './cluster-service-class';
+
+const ClusterServiceBrokerReference: K8sResourceKindReference = 'ClusterServiceBroker';
+const menuActions = Cog.factory.common;
+
+const ClusterServiceBrokerHeader: React.SFC<ClusterServiceBrokerHeaderProps> = props => <ListHeader>
+  <ColHead {...props} className="col-sm-3 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-3 col-xs-6" sortFunc="serviceCatalogStatus">Status</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="spec.relistBehavior">Relist Behavior</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="status.lastCatalogRetrievalTime">Last Retrieved</ColHead>
+</ListHeader>;
+
+const ClusterServiceBrokerListRow: React.SFC<ClusterServiceBrokerRowProps> = ({obj: serviceBroker}) => <ResourceRow obj={serviceBroker}>
+  <div className="col-sm-3 col-xs-6 co-resource-link-wrapper">
+    <ResourceCog actions={menuActions} kind="ClusterServiceBroker" resource={serviceBroker} />
+    <ResourceLink kind="ClusterServiceBroker" name={serviceBroker.metadata.name} />
+  </div>
+  <div className="col-sm-3 col-xs-6 co-break-word">
+    <StatusWithIcon obj={serviceBroker} />
+  </div>
+  <div className="col-sm-3 hidden-xs">
+    {serviceBroker.spec.relistBehavior}
+  </div>
+  <div className="col-sm-3 hidden-xs">
+    <Timestamp timestamp={serviceBroker.status.lastCatalogRetrievalTime} />
+  </div>
+</ResourceRow>;
+
+const ClusterServiceBrokerDetails: React.SFC<ClusterServiceBrokerDetailsProps> = ({obj: serviceBroker}) => {
+  return <React.Fragment>
+    <div className="co-m-pane__body">
+      <div className="row">
+        <div className="col-md-6">
+          <SectionHeading text="Service Broker Overview" />
```

Comment:
Sorry missed this before, but this should be pulled up outside of the `row` div since there is only one heading for the page.

Reply:
Ok.  Also fixed in ServiceClassDetails.